freki.go

```diff
@@ -115,6 +119,14 @@ func (p *Processor) AddServer(s Server) {
 	p.servers[s.Type()] = s
 }
 
+func (p *Processor) RegisterConnHandler(target string, handler ConnHandlerFunc) error {
+	if _, ok := p.connHandlers[target]; ok {
+		return fmt.Errorf("conn handler already registered for %s", target)
```

Comment:
This error is usually passed out to client code. I don't think the errors themselves should contain the prefix. IMO, we only use the prefix when `printf`ing the message to a logger.